pydicom/tests/test_encaps.py

```diff
@@ -0,0 +1,776 @@
+# Copyright 2008-2017 pydicom authors. See LICENSE file for details.
+"""Test for encaps.py"""
+
+import pytest
+
+from pydicom.encaps import (generate_pixel_data_fragment, get_frame_offsets,
+                            generate_pixel_data_frame, generate_pixel_data,
+                            decode_data_sequence, defragment_data, read_item)
+from pydicom.filebase import DicomBytesIO
+
+
+def assert_raises_regex(type_error, message, func, *args, **kwargs):
```

Comment:
I would move that inside a `testing.py` module since it could be used in some other test files.